home.css

```diff
@@ -0,0 +1,473 @@
+  color: white;
+}
+
+.phrasesh2 {
+  font-size: 25px;
+}
+
+.phraseh1 {
+  font-size: 34px;
+}
+
+.headParaDiv {
+  margin-left: 30px;
+  border: 1px solid red;
+  background-color: white;
+  opacity: 0.6;
+  margin-right: 30px;
+}
+
+.headPara {
+  margin-left: 5px;
+  color: red;
+}
+
+.headLoc {
+  margin-left: 30px;
+  color: white;
+}
+
+.headL {
+  font-family: "COCOGOOSE", sans-serif;
+}
+
+.headLocw {
+  margin-top: -10px;
+}
+
+.mainDiv {
+  margin-top: -27px;
+}
+
+.wedoB {
+  height: 900px;
+  background-image: url("img/wedo_back.jpg");
+}
+
+.doHead {
+  text-align: center;
+  color: red;
+  font-size: 20px;
+}
+
+.doDivLine {
+  display: grid;
+  grid-template-columns: auto;
+  grid-gap: 10px;
+  margin-top: 60px;
+}
+
+.doDiv {
+  display: flex;
+  gap: 20px;
+  justify-content: center;
+  align-items: center;
+  background-color: rgba(255, 248, 248, 0.082);
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.doDivImg {
+  margin-left: 10px;
+}
+
+.doDivHead {
+  font-size: 25px;
+}
+
+.doheadHr {
+  margin-left: 150px;
+  margin-right: 150px;
+  margin-top: -20px;
+  margin-bottom: 20px;
+}
+
+.doivpara {
+  color: white;
+  margin-left: 10px;
+  font-size: 13px;
+}
+
+.doInno {
+  background-color: red;
+  margin-left: 30px;
+  margin-right: 30px;
+  height: 80px;
+  text-align: center;
+  display: block;
+}
+
+.doLink {
+  display: none;
+}
+
+.doInnoPara {
+  margin-top: 40px;
+  padding-top: 30px;
+  font-size: 25px;
+}
+
+.ourTeam {
+  text-align: center;
+  color: black;
+  font-size: 20px;
+}
+
+.ourHr {
+  color: red;
+  margin-left: 150px;
+  margin-right: 150px;
+  margin-top: -20px;
+}
+
+.teamGrid {
+  display: grid;
+  grid-template-columns: auto;
+  grid-gap: 10px;
+}
+
+.teamDec {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid red;
+}
+
+.teamDesc {
+  color: rgb(83, 85, 87);
+  font-size: 12px;
+}
+
+.avatar1 {
+  height: 100px;
+  width: 100px;
+  margin-left: 10px;
+}
+
+.teamDet {
+  list-style-type: none;
+}
+
+.teamHr {
+  margin-right: 190px;
+}
+
+.titleT {
+  color: red;
+}
+
+footer {
+  margin-top: 100px;
+}
+
+.partners {
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(58, 56, 56);
+  padding: 10px;
+}
+
+.partnersDiv {
+  text-align: center;
+  padding: 20px 0;
+  font-size: 30px;
+}
+
+.partOn {
+  display: flex;
+  justify-content: center;
+  gap: 80px;
+}
+
+.partTw {
+  justify-content: center;
+  gap: 80px;
+  display: flex;
+}
+
+.div1 {
+  color: white;
+}
+
+.div1Hr {
+  color: red;
+  margin-left: 150px;
+  margin-right: 150px;
+}
+
+.div2 {
+  color: rgb(190, 195, 199);
+}
+
+.div4 {
+  color: rgb(142, 145, 148);
+}
+
+.div5 {
+  color: rgb(190, 195, 199);
+}
+
+.div3 {
+  color: rgb(101, 103, 104);
+}
+
+.copyDiv {
+  display: flex;
+}
+
+.copk {
+  color: rgb(83, 85, 87);
+}
+
+.copyright {
+  display: flex;
+  margin-top: 90px;
+  justify-content: center;
+  align-items: center;
+  margin-left: 20px;
+  gap: 30px;
+}
```

Comment:
- [ ] Please, be careful when you are implementing changes in your code. Sometimes It generates changes in other parts of your project as the alignment of your footer in this case. 😉 

![footer_is](https://user-images.githubusercontent.com/79876959/142710431-a29f8032-02b0-4fc1-9023-2b8f59416829.PNG)

![footer_should_be](https://user-images.githubusercontent.com/79876959/142710436-d94f58c6-971f-40d3-9bf1-6485962a93d5.PNG)